tests/unit/visual/tst_listitem.12.qml

```diff
@@ -649,7 +649,7 @@ Item {
             return [
                 {tag: "No actions", item: highlightTest, x: centerOf(highlightTest).x, y: centerOf(highlightTest).y, pressed: false},
                 {tag: "Leading/trailing actions", item: testItem, x: centerOf(testItem).x, y: centerOf(testItem).y, pressed: true},
-                {tag: "Active component content", item: controlItem, x: units.gu(1), y: units.gu(1), pressed: true},
+                {tag: "Active component content", item: controlItem, x: units.gu(1), y: units.gu(1), pressed: false},
```

Comment:
Are you sure this is correct?

Reply:
Not totally, but the test doesn't quit make sense to me logically either. However, the data item immediately following this one is using the same `item` and also has `pressed` as `false` which suggests that this change is correct.